task_1/index.js

```diff
@@ -1,10 +1,15 @@
-const onMenueItemClick = (elementId)=>{
+const onMenueItemClick = (elementId) => {
     const numberOfItem = document.getElementsByClassName('nav-items').length;
-         for(let i = 0 ; i < numberOfItem; i++){
-            if(document.getElementsByClassName('nav-items')[i].classList.contains('active')){
-                document.getElementsByClassName('nav-items')[i].classList.remove('active');
-         }
+    for (let i = 0; i < numberOfItem; i++) {
+        if (document.getElementsByClassName('nav-items')[i].classList.contains('active')) {
+            document.getElementsByClassName('nav-items')[i].classList.remove('active');
         }
```

Comment:
Use forEach instead see this example 
```suggestion
      numberOfItem.forEach((el) => {
.....})
```